lib/routes/account.js

```diff
@@ -311,6 +311,17 @@ module.exports = function (
               .then(
                 function (tokens) {
                   if (service === 'sync' && request.payload.reason === 'signin') {
+                    db.sessions(emailRecord.uid)
+                      .then(function (sessions) {
+                        log.event('login', {
+                          service: 'sync',
+                          uid: emailRecord.uid,
+                          email: emailRecord.email,
+                          isFirstDevice: sessions.length === 1,
+                          userAgent: request.headers['user-agent']
+                        })
+                      })
```

Comment:
Actually come to think of it, we'll also need to emit this event in response to `/account/create` for when the user connects their first device.  We don't send the email in that case though, since the user already gets a different suite of emails in that case.